Makefile

```diff
@@ -39,6 +39,9 @@ clean:
 	mkdir $(OUTPUT_DIR)
 	mkdir $(OUTPUT_DIR_UNMINIFIED)
 
+inline:
+	perl inliner.pl
```

Comment:
I had to go with Perl instead of simple in-place sed edit because base64 string used for replacement is too long and does not fit into `ARG_MAX`